lib/src/intl_suggestors/intl_importer.dart

```diff
@@ -124,9 +137,34 @@ _InsertionLocation _insertionLocationForPackageImport(
         trailingNewlineCount: 2);
   }
 
+  imports.forEach((importDirective) {
+    final uriContent = importDirective.uriContent;
+    if(uriContent != null){
+      if(uriContent.startsWith('package:')){
+        hasPackageImports = true;
+      } else if(uriContent.startsWith('../')){
```

Comment:
A relative import doesn't necessarily start with "..", because it could be importing at the same level, for example
import 'other_file.dart';

You can ask a URI if it's relative or absolute, which I think would be better.